src/sqllogictest/src/parser.rs

```diff
@@ -355,8 +358,25 @@ impl<'a> Parser<'a> {
         static DOUBLE_LINE_REGEX: Lazy<Regex> =
             Lazy::new(|| Regex::new(r"(\n|\r\n|$)(\n|\r\n|$)").unwrap());
         let sql = self.split_at(&QUERY_OUTPUT_REGEX)?;
-        let output_str = self.split_at(&DOUBLE_LINE_REGEX)?.trim_start();
-        let output = Output::Values(output_str.lines().map(String::from).collect());
+        static EOF_REGEX: Lazy<Regex> = Lazy::new(|| Regex::new(r"(\n|\r\n)EOF(\n|\r\n)").unwrap());
```

Comment:
We may want to hoist these regexes out of this function since they're re-defined multiple times in other functions.